content/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request.md

```diff
@@ -45,9 +45,15 @@ If you decide you don't want the changes in a topic branch to be merged to the u
 
 {% data reusables.files.choose-commit-email %}
 
+    {% note %}
+
+    **Note:** The email selector is not available for rebase merges, which do not create a merge commit, or for squash merges, which credit the user who created the pull request as the author of the squashed commit.
+
+    {% endnote %}
```

Comment:
Indented one space too many resulting in this being rendered as a `pre` element.
```suggestion
   {% note %}

   **Note:** The email selector is not available for rebase merges, which do not create a merge commit, or for squash merges, which credit the user who created the pull request as the author of the squashed commit.

   {% endnote %}
```